book/types_of_data.md

```diff
@@ -362,29 +360,83 @@ You can also create a table as a list of records, JSON-style:
 ╰───┴──────┴──────╯
 ```
 
-:::tip
-Internally, tables are simply **lists of records**. This means that any command which extracts or isolates a specific row of a table will produce a record. For example, `get 0`, when used on a list, extracts the first value. But when used on a table (a list of records), it extracts a record:
+To access a row or a column, you can use the [`get`](/commands/docs/get.md) command or cell path access:
```

Comment:
You're missing the primary explanation of what `get` does here versus `select`. `get` gets data out of its shape and transforms the data. `select` maintains the shape of the data. Say that however you think make sense, but it's a fundamental difference between `get` and `select`.